src/Domain/Creators/Brick.hpp

```diff
@@ -96,12 +109,18 @@ class Brick : public DomainCreator<3> {
   std::vector<std::array<size_t, 3>> initial_refinement_levels() const
       noexcept override;
 
+  std::unordered_map<std::string,
+                     std::unique_ptr<domain::FunctionsOfTime::FunctionOfTime>>
+  functions_of_time() const noexcept override;
```

Comment:
use the `->` syntax to avoid disrupting the dox with the long return type. (same for rectangle, interval)

Reply:
Good call! In case you're interested, it's named "trailing return type syntax" :)